### PR TITLE
I3S-Debug - Semantic Validator improvements

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
image

- Added Clear All Button
- Minor refactoring

### CODE REVIEW

**Review by belom88 — APPROVED** [review]
You could move the button to the `DebugPanel` to save space in this area:
image

**Comment by dryabinin-actionengine** [comment]
> You could move the button to the `DebugPanel` to save space in this area:
> image

Done

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 1]
```diff
@@ -315,7 +319,18 @@ export default class App extends PureComponent {
   }
 
   _renderDebugPanel() {
-    return <DebugPanel onOptionsChange={this._setDebugOptions}>{this._renderStats()}</DebugPanel>;
+    const {warnings, debugOptions} = this.state;
```
Comment (ibgreen) [inline examples/experimental/i3s-17-and-debug/app-debug.js:322]: Nit: Let's be consistent with underscores `_renderDebugPanel` vs `renderAttributesPanel`. Either no underscores or in all places.

**Review by ibgreen — APPROVED** [review]

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 1 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js:322]: Renamed.

**Pull request merged**